test/integration/functional_test.go

```diff
@@ -783,7 +815,14 @@ func validateServiceCmd(ctx context.Context, t *testing.T, profile string) {
 		}
 	}()
 
-	rr, err := Run(t, exec.CommandContext(ctx, "kubectl", "--context", profile, "create", "deployment", "hello-node", "--image=k8s.gcr.io/echoserver:1.4"))
+	var echoServerArg string
+	if arm64Platform() {
+		echoServerArg = "k8s.gcr.io/echoserver-arm:1.8"
+	} else {
+		echoServerArg = "k8s.gcr.io/echoserver:1.10"
+	}
+
+	rr, err := Run(t, exec.CommandContext(ctx, "kubectl", "--context", profile, "create", "deployment", "hello-node", "--image="+echoServerArg))
```

Comment:
how about we have to Blocks that hve the full command for each  instead of multiple ifs
```

if arm64Platform() {
// add comment why it is different
rr, err := Run(t, exec.CommandContext(ctx, "kubectl", "--context", profile, "create", "deployment", "hello-node", "--image=k8s.gcr.io/echoserver-arm:1.8"))
} else {
rr, err := Run(t, exec.CommandContext(ctx, "kubectl", "--context", profile, "create", "deployment", "hello-node", "--image=k8s.gcr.io/echoserver:1.10"))

```

and then get rid of all before logic so code be more simple to read

Reply:
done